frontend/app/_components/client-dashboard-components/SpinnerMini.js

```diff
@@ -0,0 +1,5 @@
+function SpinnerMini() {
+  return <div className='spinner-mini'></div>;
```

Comment:
where does this className 'spinner-mini' come from? It's not a tailwindCSS classname, and I didnot find any styles files for it as well.

Reply:
in global.css